src/buttons.css

```diff
@@ -194,34 +190,40 @@ against .round class */
 
 .btn.btn--pill-vc {
   border-radius: 0 !important;
-  margin-top: 1px;
-  margin-bottom: 1px;
+  margin-top: 1px !important;
+  margin-bottom: 1px !important;
+  display: block !important;
+  width: 100% !important;
 }
 
 .btn.btn--pill-vt {
   border-bottom-right-radius: 0 !important;
   border-bottom-left-radius: 0 !important;
+  display: block !important;
+  width: 100% !important;
```

Comment:
I think that these `display` and `width` properties do not need to be `!important`.

I believe our convention has been that we use `!important` when a declaration corresponds directly with the class name, and otherwise we avoid `!important` unless it's necessary. It is necessary here for the `border-radius` declarations, because we need them to be able to override `round-*` utility classes. But I don't think we need `!important` for other parts.